test/helpers/api-integration/requester.js

```diff
@@ -99,7 +99,13 @@ function _parseRes (res) {
         message: res.body.message,
       };
     }
-    return res.body.data;
+    if (Object.prototype.hasOwnProperty.call(res.body, 'data')) {
```

Comment:
Could you tell me why this change is needed? this doesn't seem to be needed for the test above or I'm missing something 😬

Reply:
Because there are almost no of positive testing in repository. I had to modify requester logic. 

```
    if (res.body.message) {
      return {
        data: res.body.data,
        message: res.body.message,
      };
    }
 ```
So far it was returning only message and status.
For this test I basically need whole response if there is any. So that's why I need to expand this class.